plugins/modules/nd_local_user.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_local_user
+version_added: "1.4.0"
+short_description: Manage local users on Cisco Nexus Dashboard
+description:
+- Manage local users on Cisco Nexus Dashboard (ND).
+- It supports creating, updating, querying, and deleting local users.
+author:
+- Gaspard Micol (@gmicol)
+options:
+  email:
+    description:
+    - The email address of the local user.
+    type: str
+  login_id:
+    description:
+    - The login ID of the local user.
+    type: str
+  first_name:
+    description:
+    - The first name of the local user.
+    type: str
+  last_name:
+    description:
+    - The last name of the local user.
+    type: str
+  user_password:
+    description:
+    - The password of the local user.
+    - Password must have a minimum of 8 characters to a maximum of 64 characters.
+    - Password must have three of the following; one number, one lower case character, one upper case character, one special character.
+    type: str
+  reuse_limitation:
+    description:
+    - The number of different passwords a user must use before they can reuse a previous one.
+    type: int
+  time_interval_limitation:
+    description:
+    - The minimum time period that must pass before a previous password can be reused.
+    type: int
+  security_domains:
+    description:
+    - The list of Security Domains and Roles for the local user.
+    type: list
+    elements: dict
+    suboptions:
+      name:
+        description:
+        - The name of the Security Domain to which give the local user access.
```

Comment:
Can we please change this to... if it's more meaningful?
- The name of the Security Domain to which the local user is given access.